website/sidebars.js

```diff
@@ -599,6 +599,23 @@ const sidebarSettings = {
         "docs/dbt-versions/upgrade-core-in-cloud",
         "docs/dbt-versions/product-lifecycles",
         "docs/dbt-versions/experimental-features",
+        {
+          type: "category",
+          label: "dbt Core upgrade guides",
+          link: {
+            type: "generated-index",
+            title: "Version upgrade guides",
+            description:
+              "Learn what's new in the latest version of dbt Core.",
+            slug: "/docs/dbt-versions/core-upgrade",
+          },
+          items: [
+            {
+              type: "autogenerated",
+              dirName: "docs/dbt-versions/core-upgrade",
+            },
+          ],
+        },
```

Comment:
@matthewshaver 
 Not sure if this is causing the weird behavior but I think we need a closing square bracket for the first items introduced in line 597
```suggestion
        },
        ],
```

Reply:
I think that closing bracket is down on line 631